platform/darwin/test/MGLStyleValueTests.swift

```diff
@@ -1,8 +1,53 @@
 import XCTest
 import Mapbox
 
-
+#if os(iOS) || os(watchOS) || os(tvOS)
+typealias ColorType = UIColor
+#elseif os(macOS)
+typealias ColorType = NSColor
+#endif
+    
 extension MGLStyleValueTests {
+    func assertColorsEqualWithAccuracy(_ actual: ColorType, _ expected: ColorType, accuracy: Float = 0.1) {
```

Comment:
Oh, whoops, I meant for that to be 0.01, although I guess 0.005 is maybe a better, as a roundish number close to 1/255.